build/build.cake

```diff
@@ -132,8 +156,72 @@ Task("NuGet-Pack")
     }
 });
 
+Task("Wait-For-Pending-Jobs")
```

Comment:
As the deployment is done via cake script, I have to make sure that previous jobs finished successfully in order to publish the package. So I need to query AppVeyor API, as there is no way of doing deployment per build (build may contain multiple jobs) without doing environment deployment - see https://help.appveyor.com/discussions/questions/35866-how-do-i-run-a-script-after-all-matrix-possibilities-succeed